README.md

```diff
@@ -30,7 +30,7 @@ test the project:
     $ ./gradlew assembleVanillaDebug # assemble the debug .apk
     $ ./gradlew installVanillaDebug  # install the debug .apk if you have an
                                      # emulator or an Android device connected
-    $ ./gradlew WordPress:cAT        # assemble, install and run unit tests
+    $ ./gradlew cAT                  # assemble, install and run unit tests
```

Comment:
Running `./gradlew cAT` resulted in the dreaded "PermGen space" error. Would info on resolving that be useful to add to this README?